_episodes/04-changes.md

```diff
@@ -637,11 +637,12 @@ repository (`git commit`):
 > > {: .bash}
 > > ~~~
 > > [master cc127c2]
-> > Wrote down my plans to start a base on venus
-> > 2 files changed, 2 insertions(+)
+> >  Wrote down my plans to start a base on venus
```

Comment:
The extra space is to replicate what the output looks like in the terminal -- at least on my machine, when I make a commit, the first line of the output (i.e., the line starting `[master cc127c2]`) is all the way over to the left, and everything after that is indented by a single space.

Example:

~~~
$ git commit -m "Initial commit"
[master (root-commit) fe775a8] Initial commit
 1 file changed, 11 insertions(+)
 create mode 100644 build-classification.py
~~~